src/main.cpp

```diff
@@ -708,8 +708,11 @@ bool	check_terminal_size(Game *game)
 {
 	int y;
 	int x;
+	long pause_time;
 
 	getmaxyx(stdscr, y, x);
+	if (shared_players_hp(game) > 0)
+		pause_time = get_current_time_in_seconds();
```

Comment:
Also, sorry that I missed it the first time, can we please remove the first if check? I feel uneasy leaving `pause_time` uninitialized.
```suggestion
	pause_time = get_current_time_in_seconds();
```
I know it's obvious atm that its use has the same condition as its initialization. But I think being defensive is almost always a good idea. One only needs to add one additional check in the first `if` without adding the same check in the second `if` to introduce a bug that can stay unnotized quite easily.

Reply:
Agreed. It's also kind of unnecessary.